lib/src/helpers/user.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:async';
+
+import 'package:rxdart/rxdart.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class UserHelper {
```

Comment:
Why is this UserHelper class present? Surely we need to get the user from AppState further up the tree

Reply:
the static methods are transferred to the user in models, 
and the appState will be lost after closing an opening the app, 
so to persist user we will save the user in sharedpreferences

Reply:
and UserHelper is deleted.
Instead of getting user from app state, user is being retrieved and strored in a account_bloc which is passed to down the tree using bloc_provider